starlite/_kwargs/extractors.py

```diff
@@ -143,7 +144,7 @@ def parse_connection_headers(connection: ASGIConnection, _: KwargsModel) -> dict
     parsed_headers = connection.scope["_headers"] = (  # type: ignore
         connection._headers if connection._headers is not Empty else parse_headers(tuple(connection.scope["headers"]))
     )
-    return cast("Dict[str, Any]", parsed_headers)
+    return cast("dict[str, Any]", parsed_headers)
```

Comment:
Why do we go from `Dict` to `dict`  here

Reply:
The annotation is a runtime string so it doesn't matter that `dict` is not subscriptable at runtime in 3.8, and static type checkers can read that as a type annotation irrespective of the runtime version.